.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "vendor/frontier"]
+	path = vendor/frontier
+	url = https://github.com/PureStake/frontier.git
+	branch = joshy-parachains-v1
```

Comment:
this branch name seems indicative of some features we may not want included, though might be for name's sake. Do we know when this ought to be merged and what its contents are?

Reply:
It got merged earlier today so I can just switch back to default frontier.

Reply:
Also note that this doesn't mean anything -- what matters is the commit hash of the vendor subdirectory, which points to our fork of frontier.